sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aadb2c/properties/AADB2CProperties.java

```diff
@@ -108,10 +100,13 @@
      */
     private String loginFlow = DEFAULT_KEY_SIGN_UP_OR_SIGN_IN;
 
+    /**
+     * User flows.
+     */
```

Comment:
Same here, more hints could be added later.

Reply:
Issue created: https://github.com/Azure/azure-sdk-for-java/issues/26600